src/tests/helpers/api.ts

```diff
@@ -149,6 +153,7 @@ export class API {
           textAlign: rest.textAlign ?? appState.currentItemTextAlign,
           verticalAlign: rest.verticalAlign ?? DEFAULT_VERTICAL_ALIGN,
           containerId: rest.containerId ?? undefined,
+          colorRanges,
```

Comment:
I think this should be something like `colorRanges: rest.colorRanges ?? {}`.

Reply:
The default of `{}` is applied inside of `newTextElement`. I think `colorRanges` should be either optional _or_ required in both `newTexrElement` _and_ `createElement` for consistency, therefore I don't think it makes sense to apply the default here and we can instead pass the `undefined` value through to be handled by `newTextElement`.

Or have I misunderstood your point?

Reply:
I was referring more to consistency with the surrounding lines of code here.  But consistency in parameters for `newTextElement` and `createElement` makes sense, where optional seems the most consistent.